util/test/paratest.server

```diff
@@ -606,6 +606,8 @@ sub main {
             $futures_mode = 2;
         } elsif (/^-futures$/) {
             $futures_mode = 1;
+        } elsif (/^-no-futures$/) {
+            $futures_mode = 0;
```

Comment:
Please add `-no-futures` to the usage and help messages (and update the help message for `-futures` to note change in defaults)

Reply:
werps!

Reply:
I wrestled with whether to alphabetize `-no-futures` with `-node` or `-futures`, and went with the former since things were pretty strictly alphabetical as they stand, and there isn't really a precedent for `-[no-]futures` as with the compiler (not to mention it seems confusing with the other square brackets in the usage), but let me know if you prefer differently.